tensorflow/compiler/xla/service/all_reduce_key.cc

```diff
@@ -30,7 +30,8 @@ std::optional<AllReduceKey> GetAllReduceKey(const HloInstruction* instruction,
     return std::nullopt;
   }
 
-  if (instruction->to_apply()->instruction_count() != 3 ||
+  if ((instruction->to_apply()->instruction_count() != 3 &&
+       instruction->to_apply()->instruction_count() != 6) ||
```

Comment:
Maybe add a comment what do 3 and 6 stand for?

Reply:
added